### DISCUSSION TITLE
openpose CN

### DISCUSSION THREAD

**Discussion by TanvirHafiz** [mashb1t/Fooocus#79] [Q&A] [answered]
Any chance that openpose style controlnet will be implemented anytime soon? some of the other forks have it. but they have removed faceswap from it. would be great to have both openpose and faceswap in a version. or you anyone knows a fork that has both please enlighten me. thanks!

**Comment by mashb1t** [comment, marked as answer]
Development of Fooocus has been paused indefinitely, see https://github.com/lllyasviel/Fooocus/discussions/3504
I'm not aware of a fork having OpenPose and FaceSwap.
